File: Programmers/num42842.py
def solution(brown, yellow):
    answer = []
    x = 0
    y = 0 
        
    for i in range (yellow, 0 , -1 ):
        x = i
        y = yellow / x
        if (x * 2) + (y * 2) + 4 == brown:
            answer.append(x+2)
            answer.append(y+2)
            break
  
    return answer

File: Programmers/num42842.java
class Solution {
    public int[] solution(int brown, int yellow) {
        int[] answer = new int[2];
        int size = brown + yellow;
         for (int i=3; i<size; i++) {
            for (int j=3; j<size; j++) {
                if (i*j == size && i>=j && (i-2) * (j-2) == yellow) {
                    answer[0] = i;
                    answer[1] = j;
                    return answer;
                }
            }
        }
        return answer;
    }
}
